OVH_BRANCH

Salut j'ai fait la MAJ ce matin.
Première erreur ma version de NODE était trop basse (8.0) je l'ai montée en 8.17 ce qui a permis l'installation de YARN sans erreur.
Cependant je n'ai pas pu redémarrer peertube que dans l'admin de yunohost et j'ai une erreur 502 bad gateway NGINX.
Une idée ? Je peux trouver des logs à te montrer ?
Merci

Bon en relisant la marche à suivre de l'install j'ai lancé 

ce qui me donne les erreurs 

>  iphodase.fr peertube[22840]: > peertube@2.0.0 start /var/www/peertube
Jan 03 08:34:24 iphodase.fr peertube[22840]: > node dist/server
Jan 03 08:34:25 iphodase.fr peertube[22840]: Something went wrong installing the "sharp" module
Jan 03 08:34:25 iphodase.fr peertube[22840]: Module did not self-register.
Jan 03 08:34:25 iphodase.fr peertube[22840]: - Remove the "node_modules/sharp" directory, run "npm install" and look for errors
Jan 03 08:34:25 iphodase.fr peertube[22840]: - Consult the installation documentation at https://sharp.pixelplumbing.com/en/stable/install/
Jan 03 08:34:25 iphodase.fr peertube[22840]: - Search for this error at https://github.com/lovell/sharp/issues
Jan 03 08:34:25 iphodase.fr peertube[22840]: npm ERR! code ELIFECYCLE
Jan 03 08:34:25 iphodase.fr peertube[22840]: npm ERR! errno 1
Jan 03 08:34:25 iphodase.fr peertube[22840]: npm ERR! peertube@2.0.0 start: `node dist/server`
Jan 03 08:34:25 iphodase.fr peertube[22840]: npm ERR! Exit status 1
Jan 03 08:34:25 iphodase.fr peertube[22840]: npm ERR!
Jan 03 08:34:25 iphodase.fr peertube[22840]: npm ERR! Failed at the peertube@2.0.0 start script.
Jan 03 08:34:25 iphodase.fr peertube[22840]: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
Jan 03 08:34:25 iphodase.fr systemd[1]: peertube.service: Main process exited, code=exited, status=1/FAILURE
Jan 03 08:34:25 iphodase.fr systemd[1]: peertube.service: Unit entered failed state.
Jan 03 08:34:25 iphodase.fr systemd[1]: peertube.service: Failed with result 'exit-code'.
Jan 03 08:34:26 iphodase.fr systemd[1]: peertube.service: Service hold-off time over, scheduling restart.
Jan 03 08:34:26 iphodase.fr systemd[1]: Stopped peertube daemon.
Jan 03 08:34:26 iphodase.fr systemd[1]: peertube.service: Start request repeated too quickly.
Jan 03 08:34:26 iphodase.fr systemd[1]: Failed to start peertube daemon.
Jan 03 08:34:26 iphodase.fr systemd[1]: peertube.service: Unit entered failed state.
Jan 03 08:34:26 iphodase.fr systemd[1]: peertube.service: Failed with result 'exit-code'.
^C

> 
Si je comprend bien c'est ce module node_modules/sharp qui pose problème.
là je coince je ne comprend pas la doc pour l'installer. Si tu as la patience ^^
Merci

> 
> 
> Bon en relisant la marche à suivre de l'install j'ai lancé
> 
> ce qui me donne les erreurs
> 
> > iphodase.fr peertube[22840]: > peertube@2.0.0 start /var/www/peertube
> > Jan 03 08:34:24 iphodase.fr peertube[22840]: > node dist/server
> > Jan 03 08:34:25 iphodase.fr peertube[22840]: Something went wrong installing the "sharp" module
> > Jan 03 08:34:25 iphodase.fr peertube[22840]: Module did not self-register.
> > Jan 03 08:34:25 iphodase.fr peertube[22840]: - Remove the "node_modules/sharp" directory, run "npm install" and look for errors
> > Jan 03 08:34:25 iphodase.fr peertube[22840]: - Consult the installation documentation at https://sharp.pixelplumbing.com/en/stable/install/
> > Jan 03 08:34:25 iphodase.fr peertube[22840]: - Search for this error at https://github.com/lovell/sharp/issues
> > Jan 03 08:34:25 iphodase.fr peertube[22840]: npm ERR! code ELIFECYCLE
> > Jan 03 08:34:25 iphodase.fr peertube[22840]: npm ERR! errno 1
> > Jan 03 08:34:25 iphodase.fr peertube[22840]: npm ERR! peertube@2.0.0 start: `node dist/server`
> > Jan 03 08:34:25 iphodase.fr peertube[22840]: npm ERR! Exit status 1
> > Jan 03 08:34:25 iphodase.fr peertube[22840]: npm ERR!
> > Jan 03 08:34:25 iphodase.fr peertube[22840]: npm ERR! Failed at the peertube@2.0.0 start script.
> > Jan 03 08:34:25 iphodase.fr peertube[22840]: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
> > Jan 03 08:34:25 iphodase.fr systemd[1]: peertube.service: Main process exited, code=exited, status=1/FAILURE
> > Jan 03 08:34:25 iphodase.fr systemd[1]: peertube.service: Unit entered failed state.
> > Jan 03 08:34:25 iphodase.fr systemd[1]: peertube.service: Failed with result 'exit-code'.
> > Jan 03 08:34:26 iphodase.fr systemd[1]: peertube.service: Service hold-off time over, scheduling restart.
> > Jan 03 08:34:26 iphodase.fr systemd[1]: Stopped peertube daemon.
> > Jan 03 08:34:26 iphodase.fr systemd[1]: peertube.service: Start request repeated too quickly.
> > Jan 03 08:34:26 iphodase.fr systemd[1]: Failed to start peertube daemon.
> > Jan 03 08:34:26 iphodase.fr systemd[1]: peertube.service: Unit entered failed state.
> > Jan 03 08:34:26 iphodase.fr systemd[1]: peertube.service: Failed with result 'exit-code'.
> > ^C
> 
> > 
> 
> Si je comprend bien c'est ce module node_modules/sharp qui pose problème.
> là je coince je ne comprend pas la doc pour l'installer. Si tu as la patience ^^
> Merci

normalement il faut juste suivre les instructions:
- supprimer le dossier node_modules/sharp `rm -rf /var/www/peertube/node_modules/sharp/`
- exécuter depuis /var/www/peertube : `sudo -u peertube env PATH=/opt/node_n/n/versions/node/10/bin:/opt/node_n/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin yarn install --production --pure-lockfile`

Merci

> code ENOENT
npm ERR! syscall open
npm ERR! path /package.json
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, open '/package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

> 

>  info it worked if it ends with ok
1 verbose cli [ '/opt/node_n/n/versions/node/10.18.1/bin/node',
1 verbose cli   '/opt/node_n/n/versions/node/10/bin/npm',
1 verbose cli   'start' ]
2 info using npm@6.13.4
3 info using node@v10.18.1
4 verbose stack Error: ENOENT: no such file or directory, open '/package.json'
5 verbose cwd /
6 verbose Linux 4.9.0-11-amd64
7 verbose argv "/opt/node_n/n/versions/node/10.18.1/bin/node" "/opt/node_n/n/versions/node/10/bin/npm" "start"
8 verbose node v10.18.1
9 verbose npm  v6.13.4
10 error code ENOENT
11 error syscall open
12 error path /package.json
13 error errno -2
14 error enoent ENOENT: no such file or directory, open '/package.json'
15 error enoent This is related to npm not being able to find a file.
16 verbose exit [ -2, true ]

> 
Merde j'ai l'impression d'être revenu en arrière avec une mauvaise version

Salut,

J'ai tout refais, relancé, le service est démarré mais erreur NGINX.
voila le résultat de journalctrl -feu peertube

> 
Jan 14 13:56:07 iphodase.fr peertube[26978]:   "process": {
Jan 14 13:56:07 iphodase.fr peertube[26978]:     "pid": 26989,
Jan 14 13:56:07 iphodase.fr peertube[26978]:     "uid": 998,
Jan 14 13:56:07 iphodase.fr peertube[26978]:     "gid": 998,
Jan 14 13:56:07 iphodase.fr peertube[26978]:     "cwd": "/var/www/peertube",
Jan 14 13:56:07 iphodase.fr peertube[26978]:     "execPath": "/opt/node_n/n/versions/node/10.18.1/bin/node",
Jan 14 13:56:07 iphodase.fr peertube[26978]:     "version": "v10.18.1",
Jan 14 13:56:07 iphodase.fr peertube[26978]:     "argv": [
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "/opt/node_n/n/versions/node/10.18.1/bin/node",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "/var/www/peertube/dist/server"
Jan 14 13:56:07 iphodase.fr peertube[26978]:     ],
Jan 14 13:56:07 iphodase.fr peertube[26978]:     "memoryUsage": {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "rss": 131592192,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "heapTotal": 90574848,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "heapUsed": 50830744,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "external": 189848
Jan 14 13:56:07 iphodase.fr peertube[26978]:     }
Jan 14 13:56:07 iphodase.fr peertube[26978]:   },
Jan 14 13:56:07 iphodase.fr peertube[26978]:   "os": {
Jan 14 13:56:07 iphodase.fr peertube[26978]:     "loadavg": [
Jan 14 13:56:07 iphodase.fr peertube[26978]:       0.57763671875,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       0.75830078125,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       0.765625
Jan 14 13:56:07 iphodase.fr peertube[26978]:     ],
Jan 14 13:56:07 iphodase.fr peertube[26978]:     "uptime": 1044673
Jan 14 13:56:07 iphodase.fr peertube[26978]:   },
Jan 14 13:56:07 iphodase.fr peertube[26978]:   "trace": [
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 18,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "Module._extensions..node",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 807,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": ".node",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 32,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "Module.load",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 653,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": "load",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 12,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "tryModuleLoad",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 593,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": null,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 3,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "Module._load",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 585,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": "_load",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 17,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "Module.require",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 692,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": "require",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 18,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/helpers.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "require",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 25,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": null,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 16,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "/var/www/peertube/node_modules/bcrypt/bcrypt.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": null,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 6,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": null,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 30,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "Module._compile",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 778,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": "_compile",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 10,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "Module._extensions..js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 789,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": ".js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 32,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "Module.load",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 653,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": "load",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 12,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "tryModuleLoad",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 593,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": null,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 3,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "Module._load",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 585,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": "_load",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 17,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "Module.require",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 692,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": "require",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 18,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/helpers.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "require",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 25,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": null,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 16,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "/var/www/peertube/dist/server/helpers/peertube-crypto.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": null,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 19,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": null,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 30,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "Module._compile",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 778,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": "_compile",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 10,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "Module._extensions..js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 789,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": ".js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 32,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "Module.load",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 653,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": "load",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 12,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "tryModuleLoad",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 593,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": null,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 3,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "Module._load",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 585,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": "_load",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 17,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "Module.require",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 692,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": "require",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 18,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/helpers.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "require",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 25,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": null,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 27,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "/var/www/peertube/dist/server/models/account/user.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": null,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 26,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": null,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 30,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "Module._compile",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 778,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": "_compile",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 10,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "Module._extensions..js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 789,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": ".js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 32,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "Module.load",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 653,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": "load",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 12,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "tryModuleLoad",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 593,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": null,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 3,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "Module._load",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 585,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": "_load",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 17,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "Module.require",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 692,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": "require",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 18,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/helpers.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "require",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 25,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": null,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 16,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "/var/www/peertube/dist/server/initializers/checker-after-init.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": null,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 14,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": null,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 30,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "Module._compile",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 778,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": "_compile",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 10,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "Module._extensions..js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 789,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": ".js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 32,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "Module.load",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 653,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": "load",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 12,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "tryModuleLoad",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 593,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": null,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 3,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "Module._load",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 585,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": "_load",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 17,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "Module.require",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 692,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": "require",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 18,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/helpers.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "require",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 25,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": null,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 30,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "/var/www/peertube/dist/server.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": null,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 45,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": null,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     },
Jan 14 13:56:07 iphodase.fr peertube[26978]:     {
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "column": 30,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "file": "internal/modules/cjs/loader.js",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "function": "Module._compile",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "line": 778,
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "method": "_compile",
Jan 14 13:56:07 iphodase.fr peertube[26978]:       "native": false
Jan 14 13:56:07 iphodase.fr peertube[26978]:     }
Jan 14 13:56:07 iphodase.fr peertube[26978]:   ]
Jan 14 13:56:07 iphodase.fr peertube[26978]: }
Jan 14 13:56:10 iphodase.fr peertube[26978]: npm ERR! code ELIFECYCLE
Jan 14 13:56:10 iphodase.fr peertube[26978]: npm ERR! errno 1
Jan 14 13:56:10 iphodase.fr peertube[26978]: npm ERR! peertube@2.0.0 start: `node dist/server`
Jan 14 13:56:10 iphodase.fr peertube[26978]: npm ERR! Exit status 1
Jan 14 13:56:10 iphodase.fr peertube[26978]: npm ERR!
Jan 14 13:56:10 iphodase.fr peertube[26978]: npm ERR! Failed at the peertube@2.0.0 start script.
Jan 14 13:56:10 iphodase.fr peertube[26978]: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
Jan 14 13:56:10 iphodase.fr peertube[26978]: npm ERR! A complete log of this run can be found in:
Jan 14 13:56:10 iphodase.fr peertube[26978]: npm ERR!     /home/peertube/.npm/_logs/2020-01-14T13_56_10_791Z-debug.log
Jan 14 13:56:10 iphodase.fr systemd[1]: peertube.service: Main process exited, code=exited, status=1/FAILURE
Jan 14 13:56:10 iphodase.fr systemd[1]: peertube.service: Unit entered failed state.
Jan 14 13:56:10 iphodase.fr systemd[1]: peertube.service: Failed with result 'exit-code'.
Jan 14 13:56:11 iphodase.fr systemd[1]: peertube.service: Service hold-off time over, scheduling restart.
Jan 14 13:56:11 iphodase.fr systemd[1]: Stopped peertube daemon.
Jan 14 13:56:11 iphodase.fr systemd[1]: Started peertube daemon.
Jan 14 13:56:11 iphodase.fr peertube[27011]: > peertube@2.0.0 start /var/www/peertube
Jan 14 13:56:11 iphodase.fr peertube[27011]: > node dist/server
^C

>

--- a/README.md
+++ b/README.md
@@ -60,7 +60,7 @@ $ yunohost app upgrade peertube -u https://github.com/YunoHost-Apps/peertube_ynh
 2. After upgrade is complete run **yarn install**.
 
 ```
-$ cd /var/www/peertube && yarn install --production --pure-lockfile
+$ cd /var/www/peertube && sudo yarn install --production --pure-lockfile
 ```
 
 3. Give proper **permissions** to peertube

